src/index.jsx

```diff
@@ -16,7 +16,6 @@ if (apiClient.ensurePublicOrAuthencationAndCookies(window.location.pathname)) {
 
   ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
 
-  // identify user for future analytics calls
-  // TODO: Call before each page call.
   identifyUser();
+  window.analytics.page();
```

Comment:
Heh.  Simple enough.  I think the idea was to hide Segment inside analytics.  So, you should just add a light wrapper (like `identifyUser`) and call it here.  Maybe named `pageEvent` (unless you have a better idea)?